pkg/validators/validate_current_csv.go

```diff
@@ -7,24 +7,25 @@ import (
 	"github.com/operator-framework/operator-registry/pkg/registry"
 )
 
-func ValidateCSVPresent(metabundle utils.MetaBundle) (bool, error) {
+// TODO - review failureMsg ++ missing test_validate_csv testing bundle
+func ValidateCSVPresent(metabundle utils.MetaBundle) (bool, string, error) {
 	if len(metabundle.Bundles) == 0 {
-		return false, fmt.Errorf("No bundles present!")
+		return false, "", fmt.Errorf("No bundles present!")
```

Comment:
```suggestion
		return false, "No bundles present", nil
```
Shouldn't it be this, because errors were meant to be indicative of unexpected bugs/errs ?

Reply:
The way I understand our interface is this:
> // ValidateFunc - returns a triple consisting of:
> // 1. bool       - true if MetaBundle validation was successful
> // 2. failureMsg - "" if the result was true, else information about why the validation failed
> // 3. error      - report any error that happened in the validation code

Reply:
Basically, we want to distinguish code error from validation error.

Reply:
EDIT:

you are right , fixed it ! Sorry brain fart